ruby/red-arrow/lib/arrow/table.rb

```diff
@@ -426,6 +426,14 @@ def remove_column(name_or_index)
       remove_column_raw(index)
     end
 
+    # Return column names in an Array.
+    #
+    # @return [::Array<String>] column names.
+    # 
+    def column_names
+      columns.map(&:name)
+    end
```

Comment:
How about defining this in `Arrow::ColumnContainable`?

Reply:
I cannot find a good place. Thank you.